src/parse.rs

```diff
@@ -99,7 +99,7 @@ impl<'a, 'b> Parse<(codemap::Span, &'a mut base::ExtCtxt<'b>, Option<ast::Ident>
             //     - describe!
             //
             // Any other top-level idents are not allowed.
-            let block_name = parser.parse_ident().ok().unwrap();
+            let block_name = format!("{}", parser.parse_ident().ok().unwrap().name);
 
             match block_name.as_str() {
```

Comment:
Ah, yes! :+1: I got as far as `*` but somehow didn't make the extra step to `&*`.